doc/index.rst

```diff
@@ -1,28 +1,19 @@
 .. _index:
 
-Ubuntu Installation documentation
-#################################
+Ubuntu installation guide
+=========================
 
-A single sentence that says what the product is, succinctly and memorably
-consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-dolore magna aliqua.
+Subiquity is an installer framework for Ubuntu. It provides the Ubuntu Server with a text-based installation user interface and Ubuntu Core first-boot configuration.
```

Comment:
Subiquity is the backend for ubuntu-desktop-installer, a short mention here sounds wise.

Reply:
Ack